conda-recipe/meta.yaml

```diff
@@ -35,6 +20,12 @@ requirements:
     - python
     - pandas
 
+test:
+  requires:
+    - cycamore ==0.0
+    - pyne
+    - nose
+
 about:
-  home: Cymetric
+  home: Cyclus
```

Comment:
Cycamore had Cyclus here so I thought it was supposed to be the project name of Cyclus